cishouseholds/edit.py

```diff
@@ -22,6 +22,18 @@ def convert_columns_to_timestamps(df: DataFrame, column_format_map: dict) -> Dat
     return df
 
 
+def format_string_upper_and_clean(df: DataFrame, column_name_to_assign: str) -> str:
```

Comment:
i appreciate this function is small and self-explanatory but should we stick to the Numpy docstring formatting?